restapi/restapi.go

```diff
@@ -430,19 +428,32 @@ func (r *RestAPI) getSalesforceOauthToken() (string, error) {
 	return authResponse.AccessToken, nil
 }
 
+func (r *RestAPI) getTimeout() int {
+	timeout := r.HttpTimeoutSeconds
+	if timeout < 1 || timeout > 600 { // don't allow timeouts less than a second or more than 10 minutes
+		timeout = DefaultHttpTimeoutSeconds
+	}
+	return timeout
+}
+
+func (r *RestAPI) logHttpTimeout() {
+	timeout := r.getTimeout()
+	timeoutMessage := "RestAPI timeout in seconds defaults to %d.  Configured value: %d."
+	log.Printf(timeoutMessage, DefaultHttpTimeoutSeconds, timeout)
```

Comment:
Since `timeoutMessage` is only used once now, it would be better to not create a variable for it.